upi/vsphere/stage2/9.post-deployment/playbooks/object_storage.yaml

```diff
@@ -0,0 +1,32 @@
+---
+- hosts: localhost
+
+  tasks:
+  - name: Create secret with s3 access key and secret key
+    k8s:
+      state: present
+      definition:
+        apiVersion: v1
+        data:
+          REGISTRY_STORAGE_S3_ACCESSKEY: "{{ objectstorage_accesskey | b64encode }}"
+          REGISTRY_STORAGE_S3_SECRETKEY: "{{ objectstorage_secretkey | b64encode }}"
+        kind: Secret
+        metadata:
+          name: image-registry-private-configuration-user
+          namespace: openshift-image-registry
+        type: Opaque
+
+  - name: Patch s3 details into image registry
+    k8s:
+      state: present
+      definition:
+        apiVersion: imageregistry.operator.openshift.io/v1
+        kind: Config
+        metadata:
+          name: cluster
+        spec:
+          storage:
+            s3:
+              bucket: "{{ objectstorage_bucketname }}"
+              region: "us-east1"
+              regionEndpoint: "{{ objectstorage_regionendpoint }}"
```

Comment:
This needs an extra task to set the config back to Managed - for reasons I don't totally understand, making any change to the registry config makes it go to `  managementState: Removed`


Should be: 
```apiVersion: imageregistry.operator.openshift.io/v1
kind: Config
metadata:
  name: cluster
spec:
...
  managementState: Managed
...```